### PR TITLE
Update Database tests to run against the emulator

### PR DESCRIPTION
**Pull request by schmidt-sebastian**

### CODE REVIEW

**Comment by schmidt-sebastian** [comment]
@paulb777 It seems like the Database tests are not part of the CI runs.

**Comment by paulb777** [comment]
@schmidt-sebastian Database has not yet been migrated to GHA from travis

File: scripts/build.sh [hunk 1]
```diff
@@ -414,16 +418,14 @@ case "$product-$platform-$method" in
...
-        test
-      fi
+    # Integration tests are only run on iOS to minimize flake failures.
+    RunXcodebuild \
+      -workspace 'gen/FirebaseDatabase/FirebaseDatabase.xcworkspace' \
+      -scheme "FirebaseDatabase-Unit-integration" \
+      "${ios_flags[@]}" \
+      "${xcb_flags[@]}" \
+      build \
+	  test
```
Comment (paulb777) [inline scripts/build.sh:451]: nit: indent

Comment (schmidt-sebastian) [reply scripts/build.sh]: Done

**Pull request merged**